package.json

```diff
@@ -17,9 +17,11 @@
   "scripts": {
     "lint": "eslint .",
     "precommit": "lint-staged",
-    "pretest": "npm install",
-    "test": "grunt test && npm run-script test-typescript",
-    "test-typescript": "node_modules/typescript/bin/tsc --noEmit --noImplicitAny typescript/raven-tests.ts"
+    "publish": "npm run test && grunt publish",
+    "test": "npm run lint && grunt build.test && npm run test:unit && npm run test:integration && npm run test:typescript",
+    "test:unit": "mocha-chrome test/index.html",
+    "test:integration": "mocha-chrome test/integration/index.html --chrome-flags '[\"--disable-web-security\"]' --ignore-resource-errors --ignore-exceptions",
+    "test:typescript": "tsc --noEmit --noImplicitAny typescript/raven-tests.ts"
```

Comment:
I generally link linking to the version found in `node_modules` because collaborators often don't have `node_modules/.bin` on their `PATH`.

Reply:
I think when you run things using yarn/npm, it'll add `node_modules/.bin` to search path.

Reply:
Ah